applications/concept-cat/src/main/java/no/ccat/dto/ConceptPublisherMessage.java

```diff
@@ -0,0 +1,23 @@
+package no.ccat.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@EqualsAndHashCode(callSuper = false)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+public class ConceptPublisherMessage extends HarvestTriggerMessage {
```

Comment:
This can essentially be a super class and all the other message types could extend from it, can't they?

Reply:
It wasn't a good idea to begin with since Jackson can't map to base classes. Introduced JsonNode